charts/delphai-deployment/values.yaml

```diff
@@ -144,19 +144,24 @@ readinessProbe:
 ingress:
   enabled: false
 
-  # Define subdomain (regex) for HTTP (Helm template)
+  # Define subdomain (regex) for HTTP (evaluated as a Helm template)
   subdomain: 'app{{ if eq $.clusterValues.environment "review" }}.*{{ end }}'
-  # Define pefix for HTTP (Helm template)
+  # Define a prefix for HTTP (evaluated as a Helm template)
   urlPrefix: "/service/{{ $.Release.Name }}/"
 
+  # Enable exposing public client-facing API
+  publicApiEnabled: false
+  # Define a prefix (regex) for the public client-facing API
+  publicApiPrefix: null
```

Comment:
Do we need a separate flag for that? Won't the prefix be enough — enabled if set, disabled otherwise

But I'm not sure what's the best approach here

Reply:
Yeah, it generates nonsense when these two aren't aligned.
So maybe have one variable only to avoid this problem:
```
$ kubectl get mappings.getambassador.io
NAME                                           SOURCE HOST                                                 SOURCE PREFIX                                       DEST SERVICE                                   STATE   REASON
antons-test                                    ^app.*\.delphai.pink$                                       /service/antons-test/                               antons-test:7070                                       
antons-test-api                                api.delphai.pink                                            /antons-test%!s(<nil>)                              antons-test:7070                                       
```

Reply:
This is the first parameter that strictly requires inobvious slashes (`urlPrefix` was similar but it was only used in few places).
I think we should validate that and crash explicitly otherwise

Now it easily produces that:
```
$ kubectl get mappings.getambassador.io
NAME                                           SOURCE HOST                                                 SOURCE PREFIX                                       DEST SERVICE                                   STATE   REASON
antons-test-api                                api.delphai.pink                                            companies                                           antons-test:7070    
```

```
$ kubectl get mappings.getambassador.io
NAME                                           SOURCE HOST                                                 SOURCE PREFIX                                       DEST SERVICE                                   STATE   REASON
antons-test-api                                api.delphai.pink                                            /companies                                          antons-test:7070                                       
```

Reply:
For non-release:
```
antons-test-api                                api.delphai.pink                                            /antons-testcompanies/                              antons-test:7070                                       
```

Reply:
I scrapped the `publicApiEnabled` flag in 
062f1a27a285ab452cc6aefe77c0884923a76b10.